bigquery/google/cloud/bigquery/client.py

```diff
@@ -187,6 +187,35 @@ def job_from_resource(self, resource):
             return QueryJob.from_api_repr(resource, self)
         raise ValueError('Cannot parse job resource')
 
+    def get_job(self, job_name, project=None):
```

Comment:
Nit: I'd prefer `job_id` over `job_name` to be consistent with the terminology in the docs & API reference.